components/form/events.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Events
+page_title: Form Events
+description: Form for Blazor - Events.
+slug: form-events
+tags: telerik,blazor,form,edit,events
+published: True
+position: 25
+---
+
+# Form Events
+
+The Form component for Blazor exposes events that allow you to respond to user actions and provide custom logic.
+
+* [OnSubmit](#onsubmit)
+* [OnValidSubmit](#onvalidsubmit)
+* [OnInvalidSubmit](#oninvalidsubmit)
+
```

Comment:
Perhaps add a sentence here that the examples in this article use edit context but you don't have to, you can simply pass a model to the form

Reply:
done